cmd/kube-apiserver/app/server.go

```diff
@@ -572,11 +572,13 @@ func BuildAuthenticator(s *options.ServerRunOptions, storageFactory serverstorag
 	if client == nil || reflect.ValueOf(client).IsNil() {
 		// TODO: Remove check once client can never be nil.
 		glog.Errorf("Failed to setup bootstrap token authenticator because the loopback clientset was not setup properly.")
-	} else {
+	}
+	if authenticatorConfig.BootstrapToken {
```

Comment:
reverted, if ```authenticatorConfig.BootstrapToken``` is false（we donot setup bootstrap token）, client is nil, we will alse get this error log,
```Failed to setup bootstrap token authenticator because the loopback clientset was not setup properly.```

Reply:
and I do not see the client this parameter is used